app/controllers/hyrax/admin/admin_sets_controller.rb

```diff
@@ -22,10 +22,9 @@ class Admin::AdminSetsController < ApplicationController
     self.admin_set_create_service = AdminSetCreateService
 
     def show
-      add_breadcrumb t(:'hyrax.controls.home'), root_path
-      add_breadcrumb t(:'hyrax.dashboard.breadcrumbs.admin'), hyrax.dashboard_path
-      add_breadcrumb t(:'hyrax.admin.sidebar.admin_sets'), hyrax.admin_admin_sets_path
-      add_breadcrumb t(:'hyrax.admin.admin_sets.show.breadcrumb'), request.path
+      add_breadcrumb t(:'hyrax.dashboard.breadcrumbs.dashboard'), hyrax.dashboard_path
+      add_breadcrumb t('hyrax.dashboard.my.collections'), hyrax.my_collections_path
+      add_breadcrumb "<em>#{@admin_set.title.first}</em>", "#"
```

Comment:
Have you tested that this doesn't get escaped as it's not HTML safe?

Reply:
Thanks @jcoyne! I added .html_safe and rubocop complained, so I removed it.

Reply:
Collections use this for the show page final breadcrumb...

```
add_breadcrumb presenter.to_s, polymorphic_path(presenter)
```

And works uses this, which is basically the same...
```
add_breadcrumb presenter.to_s, main_app.polymorphic_path(presenter)
```

Reply:
Resolved